include/ginkgo/core/matrix/dense.hpp

```diff
@@ -1215,6 +1236,14 @@ class Dense
      */
     virtual void compute_squared_norm2_impl(LinOp* result) const;
 
+    /**
+     * @copydoc compute_mean(LinOp*) const
+     *
+     * @deprecated  This function will be removed in the future,
```

Comment:
Is that comment correct?

Reply:
Ah, I copy/pasted it and didn't change it. However, what are we trying to convey with the deprecated note here?

Reply:
I think there was a switch between cublas/etc and ginkgo implementation for dot products or similar.